acceptance/tests/windows/PUP-9719_windows_system_first_pa_run.rb

```diff
@@ -12,16 +12,23 @@
 
   agents.each do |agent|
     statedir = on(agent, puppet('config print statedir')).stdout.chomp
```

Comment:
I think `statedir` related code can be removed from this test, as it was only used to check for the `last_run_summary` file which is now in `publicdir`... but on the other hand this test seems finicky and I don't really want to mess with it. So, up to you.